linera-core/src/client/mod.rs

```diff
@@ -1861,18 +1854,8 @@ impl<Env: Environment> ChainClient<Env> {
     #[instrument(level = "trace")]
     pub async fn request_leader_timeout(&self) -> Result<TimeoutCertificate, ChainClientError> {
         let chain_id = self.chain_id;
-        let query = ChainInfoQuery::new(chain_id).with_committees();
-        let info = self
-            .client
-            .local_node
-            .handle_chain_info_query(query)
-            .await?
-            .info;
-        let committee = info
-            .requested_committees
-            .ok_or(LocalNodeError::InvalidChainInfoResponse)?
-            .remove(&info.epoch)
-            .ok_or(LocalNodeError::InactiveChain(chain_id))?;
+        let info = self.chain_info_with_committees().await?;
+        let committee = info.current_committee()?;
```

Comment:
I think I'd just chain `current_comittee()` call after `.await?`- w/o the intermediate `info` 🤔 

Here and below.

Reply:
I just think if `info` not used anywhere, these two are now so short and self-describing I wouldn't add more variables.

Reply:
I'll change the other call site where it's possible, but at this one here it isn't: We're using the info in the next two lines, too.